test/workspaces_test.py

```diff
@@ -53,6 +53,10 @@ def test_workspaces(self):
         all_ws = self._api.workspaces.list_all(include=["organization"])
         self.assertIn("included", all_ws)
 
+        # Test the search parameter on list all workspaces parameter
+        search_listed_ws = self._api.workspaces.list_all(page=PAGE_START, page_size=PAGE_SIZE, search=ws_name)["data"]
+        self.assertTrue(len(search_listed_ws), 1)
```

Comment:
Just to make sure we are getting the workspace we are searching for, I think we should compare strings, instead of the length of the returned values for this one.

Reply:
Updated the test to include:
`self.assertTrue(ws_name in [ws["attributes"]["name"] for ws in search_listed_ws])`

Which validates the list includes the searched workspace using the name